musicThisWeek/GeoIP2/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
```

Comment:
I propose that we keep the GeoIP2 database in the _private folder, since that's the folder I've created to house files that should not be checked into Git. This works too of course, but I think it might be nice to just keep em together. What do you think?